.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: minimal
+language: generic
```

Comment:
If this is really the case, that's quite surprising. Is travis really making people specify the wrong language to get more disk space?